README.md

```diff
@@ -101,10 +101,6 @@ spec:
 
       WORKDIR /build/
 
-      RUN yum -y install git make sudo gcc \
```

Comment:
maybe we should clarify that customer may install some proprietary package which might be need for a specific compilation, but in that case he needs  to provide the yum repo

Reply:
Updated.

Also changed the way to retrieve the DTK image from the OCP release.